test/TestSites.Portable/web.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0"?>
+<configuration>
+  <system.webServer>
+    <!-- This repository cannot use the iis publish tool because there's a bug in the installer if the tool package is a package reference -->
```

Comment:
nit: "publish-iis tool" instead of "iis publish tool"
nit: the bug is that cli does not support resolving commands in if they are not packages